pkgs/applications/audio/midisnoop/default.nix

```
@@ -0,0 +1,70 @@
+{
+  pkgs ? import <nixpkgs> { },
```

First of all, package recipes aren't written like that.
```suggestion
  stdenv,
  lib,
  fetchFromGitHub,
  rtmidi,
  python3,
  pkg-config,
```